lib/language.js

```diff
@@ -1366,7 +1382,20 @@ function init() {
       ,fi: 'Raporttia ei voida luoda liian vähäisen tiedon vuoksi. Valitse useampia päiviä.'
       ,nb: 'Der er ikke nok data til å lage rapporten.  Velg flere dager.'
       ,he: 'לא נמצא מספיק מידע ליצירת הדוח. בחר ימים נוספים.'
-	  }
+      }
+// Profile editor
+    ,'Profile editor' : {
+      it: 'NS - Dati Personali'
+      }
+    ,'General Profile setting' : {
+      it: 'Impostazioni Generali'
+      }
+    ,'Title' : {
+      it: 'Titolo'
+      } 
+    ,'Units' : {
+      ,it: 'Unità'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241525904/issues/source?bid=2452379&fileBranchId=2935957#l1397)